_posts/2022-06-08-filesystem-integrity.adoc

```diff
@@ -0,0 +1,118 @@
+---
+layout: post
+title: "Enabling Integrity on Filesystem realms"
+date: 2022-06-08
+tags: filesystem-realm integrity
+synopsis: An overview of the new support for filesystem realm integrity verification.
+author: araskar
+link:
+---
+
+Previously, any data in the filesystem realm was not verified, this meant that anyone with access to the identity files, could tamper with the data without any way of knowing. Now filesystem integrity has been added by generating a signature through the use of a public-private key pair.
```

Comment:
s/Now filesystem integrity.../Now support for filesystem integrity checking has been added by adding signatures to identity files (or something along those lines)